docs/pipelineruns.md

```diff
@@ -55,6 +55,103 @@ following fields:
 [kubernetes-overview]:
   https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields
 
+### Specifying a pipeline
+
+Since a `PipelineRun` is an invocation of a [`Pipeline`](pipelines.md), you must sepcify
+what `Pipeline` to invoke.
+
+You can do this by providing a reference to an existing `Pipeline`:
+
+```yaml
+spec:
+  pipelineRef:
+    name: myPipeline
+
+```
+
+Or you can embed the spec of the `Pipeline` directly in the `PipelineRun`:
+
+```yaml
+spec:
+  pipelineSpec:
+    tasks:
+    - name: task1
+      taskRef:
+        name: myTask
+```
+
+A sample `PipelineRun` to display greetings while embedding the spec of the `Pipeline` directly in the `PipelineRun`: 
```

Comment:
It would be great to see a similar example in the `examples/pipelineruns` directory. This would then be run as part of our test suite.

Reply:
yup, sure, moved it under `examples/` dir.